### PR TITLE
Update Form1.cs

### PR DESCRIPTION
**Pull request by rpressman**

### CODE REVIEW

File: Definechiando/Form1.cs [hunk 1]
```diff
@@ -4,7 +4,8 @@ public partial class Form1 : Form
     {
         public Form1()
         {
+            // Initializes
```
Comment (mordechaigellis) [inline Definechiando/Form1.cs:7]: please add some implementation code

File: Definechiando/Form1.cs [hunk 2]
```diff
@@ -4,7 +4,8 @@ public partial class Form1 : Form
...
             InitializeComponent();
         }
     }
-}
\ No newline at end of file
+}
```
Comment (mordechaigellis) [inline Definechiando/Form1.cs:14]: add documentation at this point

**Review by mordechaigellis — CHANGES_REQUESTED** [review]

**Comment by mordechaigellis** [comment]
I reviewed code, pretty good, please see min file comments

File: Definechiando/Form1.cs (hunk 1 above)
Comment (rpressman) [reply Definechiando/Form1.cs:7]: Added implementation

File: Definechiando/Form1.cs (hunk 2 above)
Comment (rpressman) [reply Definechiando/Form1.cs:14]: Added documentation

File: Definechiando/Form1.cs (hunk 1 above)
Comment (mordechaigellis) [reply Definechiando/Form1.cs:7]: this is good

File: Definechiando/Form1.cs (hunk 2 above)
Comment (mordechaigellis) [reply Definechiando/Form1.cs:14]: this is also good

File: Definechiando/Form1.cs [hunk 3]
```diff
@@ -4,7 +4,10 @@ public partial class Form1 : Form
...
         public Form1()
         {
+            // Initializes
+            // Implementation
             InitializeComponent();
         }
     }
-}
\ No newline at end of file
+    // Documentation
```
Comment (mordechaigellis) [inline Definechiando/Form1.cs:12]: please add more detail to the documentation

**Review by mordechaigellis — CHANGES_REQUESTED** [review]

**Comment by mordechaigellis** [comment]
I approved changes, but still need some more tweaks

File: Definechiando/Form1.cs (hunk 3 above)
Comment (rpressman) [reply Definechiando/Form1.cs:12]: Done

Comment (mordechaigellis) [reply Definechiando/Form1.cs:12]: all good

**Pull request closed without merging**